### PR TITLE
Add changelog for release v0.38.0

### PR DESCRIPTION
**Pull request by Ajarmar**
> [!warning]
> **This is a public repository, ensure not to disclose:**
>
> - [ ] personal data beyond what is necessary for interacting with this pull request, nor
> - [ ] business confidential information, such as customer names.

### What kind of PR is this?

**Required**: Mark one of the following that is applicable:

- [ ] kind/feature       
- [ ] kind/improvement   
- [ ] kind/deprecation   
- [ ] kind/documentation 
- [ ] kind/clean-up      
- [ ] kind/bug           
- [x] kind/other         

*Optional*: Mark one or more of the following that are applicable:

> [!important]
> Breaking changes should be marked `kind/admin-change` or `kind/dev-change` depending on type
> Critical security fixes should be marked with `kind/security`

- [ ] kind/admin-change 
- [ ] kind/dev-change   
- [ ] kind/security     
- [ ] [kind/adr]()      

### What does this PR do / why do we need this PR?

...

- Fixes #

#### Checklist

- [ ] Proper commit message prefix on all commits
  
- Change checks:
  - [ ] The change is transparent
  - [ ] The change is disruptive
  - [ ] The change requires no migration steps
  - [ ] The change requires migration steps
  - [ ] The change upgrades CRDs
- Metrics checks:
  - [ ] The metrics are still exposed and present in Grafana after the change
  - [ ] The metrics names didn't change (Grafana dashboards and Prometheus alerts are not affected)
  - [ ] The metrics names did change (Grafana dashboards and Prometheus alerts were fixed)
- Logs checks:
  - [ ] The logs do not show any errors after the change
- Pod Security Policy checks:
  - [ ] Any changed pod is covered by Pod Security Admission
  - [ ] Any changed pod is covered by Gatekeeper Pod Security Policies
  - [ ] The change does not cause any pods to be blocked by Pod Security Admission or Policies
- Network Policy checks:
  - [ ] Any changed pod is covered by Network Policies
  - [ ] The change does not cause any dropped packets in the `NetworkPolicy Dashboard`
- Audit checks:
  - [ ] The change does not cause any unnecessary Kubernetes audit events
  - [ ] The change requires changes to Kubernetes audit policy
- Falco checks:
  - [ ] The change does not cause any alerts to be generated by Falco
- Bug checks:
  - [ ] The bug fix is covered by regression tests
- Config checks:
  - [ ] The schema was updated

### CODE REVIEW

File: changelog/0.38.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+## Changes by kind
+
+### Feature(s)
+
+- [#1943](https://github.com/elastisys/compliantkubernetes-apps/pull/1943) - Add tekton-pipelines controller `Ayoub Ed-dafali`
+- [#1999](https://github.com/elastisys/compliantkubernetes-apps/pull/1999) - add azure storage support to apps [@vomba](https://github.com/vomba)
+- [#2013](https://github.com/elastisys/compliantkubernetes-apps/pull/2013) - Add rclone restore helm chart [@aarnq](https://github.com/aarnq)
+- [#2060](https://github.com/elastisys/compliantkubernetes-apps/pull/2060) - Support Azure DNS and VXLAN node network policy [@vomba](https://github.com/vomba)
+- [#2078](https://github.com/elastisys/compliantkubernetes-apps/pull/2078) - Added kured alert for when nodes have required reboot for 30 days [@Elias-elastisys](https://github.com/Elias-elastisys)
+- [#2101](https://github.com/elastisys/compliantkubernetes-apps/pull/2101) - Add a gatekeeper constraint that warns application developers when they deploy with less then 3 replicas [@vomba](https://github.com/vomba)
```
Comment (aarnq) [inline changelog/0.38.md:18]:
```suggestion
- [#2101](https://github.com/elastisys/compliantkubernetes-apps/pull/2101) - Add a gatekeeper constraint that warns application developers when they deploy with less then 2 replicas [@vomba](https://github.com/vomba)
```

**Review by AlbinB97 — APPROVED** [review]
LGTM 👍

File: changelog/0.38.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+## Release highlights
+
+- Gatekeeper upgrade to v3.15.1
+
+## Changes by kind
+
+### Feature(s)
+
+- [#1943](https://github.com/elastisys/compliantkubernetes-apps/pull/1943) - Add tekton-pipelines controller `Ayoub Ed-dafali`
```
Comment (davidumea) [inline changelog/0.38.md:13]: :slightly_smiling_face: 
```suggestion
- [#1943](https://github.com/elastisys/compliantkubernetes-apps/pull/1943) - Add tekton-pipelines controller `Ayoub Ed-dafali`, [@davidumea](https://github.com/davidumea)
```

File: changelog/0.38.md (hunk 1 above)
Comment (Zash) [reply changelog/0.38.md:18]:
```suggestion
- [#2101](https://github.com/elastisys/compliantkubernetes-apps/pull/2101) - Add a gatekeeper constraint that warns application developers when they deploy with less than 2 replicas [@vomba](https://github.com/vomba)
```

**Review by davidumea — APPROVED** [review]

File: changelog/0.38.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+## Release highlights
+
+- Gatekeeper upgrade to v3.15.1
+
+## Changes by kind
+
+### Feature(s)
+
+- [#1943](https://github.com/elastisys/compliantkubernetes-apps/pull/1943) - Add tekton-pipelines controller `Ayoub Ed-dafali`, [@davidumea](https://github.com/davidumea)
+- [#1999](https://github.com/elastisys/compliantkubernetes-apps/pull/1999) - add azure storage support to apps [@vomba](https://github.com/vomba)
```
Comment (Zash) [inline changelog/0.38.md:14]:
```suggestion
- [#1999](https://github.com/elastisys/compliantkubernetes-apps/pull/1999) - Add azure storage support to apps [@vomba](https://github.com/vomba)
```

File: changelog/0.38.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+- [#2013](https://github.com/elastisys/compliantkubernetes-apps/pull/2013) - Add rclone restore helm chart [@aarnq](https://github.com/aarnq)
+- [#2060](https://github.com/elastisys/compliantkubernetes-apps/pull/2060) - Support Azure DNS and VXLAN node network policy [@vomba](https://github.com/vomba)
+- [#2078](https://github.com/elastisys/compliantkubernetes-apps/pull/2078) - Added kured alert for when nodes have required reboot for 30 days [@Elias-elastisys](https://github.com/Elias-elastisys)
```
Comment (Zash) [inline changelog/0.38.md:17]:
```suggestion
- [#2078](https://github.com/elastisys/compliantkubernetes-apps/pull/2078) - Add kured alert for when nodes have required reboot for 30 days [@Elias-elastisys](https://github.com/Elias-elastisys)
```

Consistency 😄

**Review by Zash — APPROVED** [review]

**Pull request merged**